linuxNew/jdk/suse/src/main/packaging/temurin/11/temurin-11-jdk.spec

```diff
@@ -45,9 +48,20 @@
 %global vers_arch2 ppc64le
 %global vers_arch3 s390x
 %global vers_arch4 aarch64
+%global vers_arch5 arm
 %global src_num 6
 %global sha_src_num 7
 %endif
+%ifarch armv7l
```

Comment:
I don't know if suse has the `%{arm}` macro. Please check and see if you can use the same trick as for redhat specs.

Reply:
Yes, `arm` works for suse. May not work with fedora? https://github.com/adoptium/installer/issues/390#issuecomment-979489941

Reply:
not sure we are on the same page, yet :)

Reply:
@jerboaa sorry misunderstood. Updated and  `%{arm}` marco works on Suse. Thanks!